css/components/question-card.css

```diff
@@ -0,0 +1,44 @@
+.question-card {
+  margin: 100px 80px;
+  border: 1px solid black;
+  max-width: 600px;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.question-card__answer-container {
+  margin: 20px;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.question-card__button {
+  margin: 20px;
+  padding: 5px;
+  color: var(--primary-color);
+  background-color: #fff;
+  border-radius: 6px;
+  border: 1px solid var(--primary-color);
+}
+
+.question-card__text {
+  margin: 10px;
+}
+
+.question-card__list {
+  margin: 10px;
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
hier könntest du auch mit gap arbeiten, statt jedem list item ein margin zu geben

Reply:
Should I work with rem or px then? what is the better case for rem?

Reply:
I will use rem ;)